python/mypy.ini

```diff
@@ -22,6 +22,7 @@ disallow_untyped_defs = True
 show_error_codes = True
 warn_unused_ignores = True
 warn_redundant_casts = True
+namespace_packages = True
```

Comment:
quick question, why was this required? Seems llike we aren't adding a new package for native namespace packaging - I read https://packaging.python.org/en/latest/guides/packaging-namespace-packages/#native-namespace-packages

Reply:
Without this option mypy will not pass because some of the Google / GRPC stubs are namespace packages.